src/actions/mole-game-action-creator.js

```diff
@@ -0,0 +1,21 @@
+const BaseActionCreator = require('./base-action-creator');
+
+export default class MoleGameActionCreator extends BaseActionCreator {
+  // Action types
+  static ACTIVATE_PANEL = "activate-panel";
+  static DEACTIVATE_PANEL = "deactivate-panel";
+
+  /**
+   * Signals the mole game to activate a panel, typically from a timer
+   */
+  sendActivatePanel({stripId, panelId}) {
```

Comment:
For now though we should change this to something else like "available" or something. I don't know when we'll get to fixing this and it's easier to change it now before the merge.